include/game/bases/d_dylink.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <game/framework/f_profile.hpp>
+#include <lib/egg/core/eggExpHeap.h>
+
+class DynamicModuleControlBase {
+public:
+    bool load_async();
+    bool link();
+};
+
+class dDynamicModuleControl : public DynamicModuleControlBase {
+public:
+    dDynamicModuleControl(const char *name, EGG::ExpHeap *heap);
+    virtual ~dDynamicModuleControl();
```

Comment:
With how this class is laid out, the vtable is at the wrong offset. It should be at 0xC.